prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1040,6 +1040,19 @@ static int rxr_ep_setopt(fid_t fid, int level, int optname,
 		if (ret)
 			return ret;
 		break;
+	case FI_OPT_EFA_SEND_IN_ORDER_128_BYTES:
+		/* For future use only, currently we do not support in-order 
+		transfer in send/recv */
+		return -FI_EOPNOTSUPP;
+	case FI_OPT_EFA_RMA_IN_ORDER_128_BYTES:
+#if HAVE_EFA_DATA_IN_ORDER_ALIGNED_128_BYTES
+		ret = ibv_query_qp_data_in_order(rxr_ep->base_ep.qp, 
+						IBV_WR_RDMA_WRITE,
+						IBV_QUERY_QP_DATA_IN_ORDER_ALIGNED_128_BYTES);
+		if (ret)
```

Comment:
better wrap `#if` logic in a separate function `efa_base_ep_support_write_in_order_128_bytes()` and return `true` or `false`.